diff --git a/doc/tags b/doc/tags
new file mode 100644
index 0000000..98059ec
--- /dev/null
+++ b/doc/tags
@@ -0,0 +1,8 @@
+auto-dark-mode-auto-dark-mode.nvim	auto-dark-mode.txt	/*auto-dark-mode-auto-dark-mode.nvim*
+auto-dark-mode-auto-dark-mode.nvim-⚙️-configuration	auto-dark-mode.txt	/*auto-dark-mode-auto-dark-mode.nvim-⚙️-configuration*
+auto-dark-mode-auto-dark-mode.nvim-💖-contributors	auto-dark-mode.txt	/*auto-dark-mode-auto-dark-mode.nvim-💖-contributors*
+auto-dark-mode-auto-dark-mode.nvim-📋-requirements	auto-dark-mode.txt	/*auto-dark-mode-auto-dark-mode.nvim-📋-requirements*
+auto-dark-mode-auto-dark-mode.nvim-📦-installation	auto-dark-mode.txt	/*auto-dark-mode-auto-dark-mode.nvim-📦-installation*
+auto-dark-mode-auto-dark-mode.nvim-🚀-usage	auto-dark-mode.txt	/*auto-dark-mode-auto-dark-mode.nvim-🚀-usage*
+auto-dark-mode-links	auto-dark-mode.txt	/*auto-dark-mode-links*
+auto-dark-mode-table-of-contents	auto-dark-mode.txt	/*auto-dark-mode-table-of-contents*
