diff --git a/autoload/llama.vim b/autoload/llama.vim
index d86b686..6ba9214 100644
--- a/autoload/llama.vim
+++ b/autoload/llama.vim
@@ -885,8 +885,8 @@ function! s:fim_on_stdout(hash, cache, pos_x, pos_y, is_auto, job_id, data, even
     endif
 
     " setup accept shortcuts
-    inoremap <buffer> <Tab>   <C-O>:call llama#fim_accept('full')<CR>
-    inoremap <buffer> <S-Tab> <C-O>:call llama#fim_accept('line')<CR>
+    inoremap <buffer> <C-E>   <C-O>:call llama#fim_accept('full')<CR>
+    "inoremap <buffer> <S-Tab> <C-O>:call llama#fim_accept('line')<CR>
     inoremap <buffer> <C-B>   <C-O>:call llama#fim_accept('word')<CR>
 
     let s:hint_shown = v:true
